.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  
+script:
+  - gradle build
```

Comment:
I'd rather use gradlew so that there's no gradle version compability issues. I pushed a change adding it, but I still need to fix the scripts for it.